google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java

```diff
@@ -1042,4 +1045,100 @@ public Acl blobToPublicRead(String bucketName, String blobName, long blobGenerat
     return buckets;
   }
 
+  /**
+   * Example of enabling Requester pays on a bucket.
+   */
+  public Bucket enableRequesterPays(String  bucketName) throws StorageException {
+    // [START enable_requester_pays]
+    // Instantiate a Google Cloud Storage client
+    Storage storage = StorageOptions.getDefaultInstance().getService();
+
+    // The name of the existing bucket to enable requester-paying for, e.g. "my-bucket"
+    // String bucketName = "my-bucket"
+    BucketInfo bucketInfo = BucketInfo.newBuilder(bucketName)
+        .setRequesterPays(true)
+        .build();
+
+    // Update the bucket, throws StorageException on failure
+    Bucket bucket = storage.update(bucketInfo);
+
+    System.out.println("Requester pay status for " + bucketName +": " + bucket.requesterPays());
+    // [END enable_requester_pays]
+    return bucket;
+  }
+
+  /**
+   * Example of disabling Requester pays on a bucket.
+   */
+  public Bucket disableRequesterPays(String bucketName) {
+    // [START disable_requester_pays]
+    // Instantiate a Google Cloud Storage client
+    Storage storage = StorageOptions.getDefaultInstance().getService();
+
+    // The name of the bucket to disable requester-paying for, e.g. "my-bucket"
+    // String bucketName = "my-bucket"
+    BucketInfo bucketInfo = BucketInfo.newBuilder(bucketName)
+        .setRequesterPays(false)
+        .build();
+
+    // Update the bucket, throws StorageException on failure
+    Bucket bucket = storage.update(bucketInfo);
+
+    System.out.println("Requester pays status for " + bucketName +": " + bucket.requesterPays());
+    // [END disable_requester_pays]
+    return bucket;
+  }
+
+  /**
+   * Example of retrieving Requester pays status on a bucket.
+   */
+  public Bucket getRequesterPaysStatus(String bucketName) throws StorageException {
+    // [START get_requester_pays_status]
+    // Instantiate a Google Cloud Storage client
+    Storage storage = StorageOptions.getDefaultInstance().getService();
+
+    // The name of the bucket to retrieve requester-pays status, eg. "my-bucket"
+    // String bucketName = "my-bucket"
+    // Retrieve the bucket, throws StorageException on failure
+    Bucket bucket = storage.get(bucketName);
+
+    System.out.println("Requester pays status : " + bucket.requesterPays());
+    // [END get_requester_pays_status]
+    return bucket;
+  }
+
+  /**
+   * Example of downloading a file using Requester pay.
+   */
+  public void downloadFileUsingRequesterPays(String projectId, String bucketName,
+      String srcFilename, Path destFilePath) throws IOException {
+    // [START storage_download_file_requester_pays]
+    // Instantiate a Google Cloud Storage client
+
+    // The project ID to bill
+    // String projectId = "my-billable-project-id";
```

Comment:
@vam-google : is this the commenting you referred to ? If so, expected : meant to be descriptive to users on what the variables mean.